packages/vscode-integration/src/common/glsp-vscode-connector.ts

```diff
@@ -121,6 +123,7 @@ export class GlspVscodeConnector<D extends vscode.CustomDocument = vscode.Custom
             },
             onBeforePropagateMessageToClient: (_originalMessage, processedMessage) => processedMessage,
             onBeforePropagateMessageToServer: (_originalMessage, processedMessage) => processedMessage,
+            messenger: new Messenger({ ignoreHiddenViews: false }),
```

Comment:
Creating the messenger object is cheap so it does not require any disposal so we do not care that we potentially overwrite it again with the messenger that is set in the options.